src/survey/handlebars.coffee

```diff
@@ -0,0 +1,25 @@
+{go, map, tee, reject,
+include, Type, isType, isMatch,
```

Comment:
`isMatch` isn't used below. I found the same in my own PR, from the copy-pasta. :smiley_cat: